frontend/cypress/e2e/journeys.cy.js

```diff
@@ -0,0 +1,59 @@
+describe('Journeys page', () => {
+  beforeEach(() => {
+    cy.visit('/journeys');
+  });
+
+  it('should have a title', () => {
+    cy.get('.m-journey__title h1')
+      .first()
+      .should('contain', 'Discover Journeys');
+  });
+
+  it('should there be a list of 5 journeys', () => {
+    cy.get('.m-journey__gridelement').should('have.length', 5);
+  });
+
+  it('should Ethiopia journey navigates to its correspondent page', () => {
+    cy.get('.m-journey__gridelement')
+      .first()
+      .find('h2 a[href="/journeys/1"]')
+      .should('contain', 'ETHIOPIA');
+    cy.get('.m-journey__gridelement')
+      .first()
+      .find('.btn[href="/journeys/1"]')
+      .should('contain', 'Learn more');
+  });
+
+  it('should India journey navigates to its correspondent page', () => {
```

Comment:
You forgot a test for `/journeys/2`, right?

Reply:
I can spend more time doing more tests but I wanted to start the Next migration asap. I'll include it but it will be a simple test